includes/admin/reporting/class-gateway-error-logs-list-table.php

```diff
@@ -54,7 +54,7 @@ protected function get_primary_column_name() {
 	public function column_default( $item, $column_name ) {
 		switch ( $column_name ) {
 			case 'error' :
-				return get_the_title( $item['ID'] ) ? get_the_title( $item['ID'] ) : __( 'Payment Error', 'easy-digital-downloads' );
+				return $item['error'];
```

Comment:
While we're here, can you add this:

```suggestion
				return $item['error'];
			case 'payment_id' :
				return ! empty( $item['payment_id'] ) ? $item['payment_id'] : '&ndash;';
```

I think that looks better when you have a gateway error that doesn't have an associated payment ID. 

Before:

![Screenshot from 2021-01-19 13-33-28](https://user-images.githubusercontent.com/6324272/105041453-ef1ace80-5a5a-11eb-93f9-452110f9fef8.png)

After:

![Screenshot from 2021-01-19 13-33-16](https://user-images.githubusercontent.com/6324272/105041474-f2ae5580-5a5a-11eb-8906-5a5c52e5f070.png)